pkg/controller/hyperconverged/hyperconverged_controller_test.go

```diff
@@ -494,9 +526,9 @@ var _ = Describe("HyperconvergedController", func() {
 					Expect(requeue).To(BeFalse())
 					checkAvailability(foundResource, metav1.ConditionTrue)
 				})
-				By("Check KV", func() {
+				By("Check SSP", func() {
```

Comment:
What is the reason for this change?

Reply:
because the test is checking ssp not kubevirt